sdcm/utils/cloud_monitor/resources/static_ips.py

```diff
@@ -43,13 +43,13 @@ def __init__(self, eip, region):
 
 
 class GceStaticIP(StaticIP):  # pylint: disable=too-few-public-methods
-    def __init__(self, static_ip: GCEAddress):
-        used_by = static_ip.extra.get("users")
+    def __init__(self, static_ip: GceAddress):
+        used_by = static_ip.users
         super().__init__(
             cloud="gce",
             name=static_ip.name,
             address=static_ip.address,
-            region=static_ip.region if isinstance(static_ip.region, str) else static_ip.region.name,
+            region=static_ip.region if isinstance(static_ip.region, str) else static_ip.region,
```

Comment:
The condition here is useless because we have identical values on both sides.
Please either remove the condition or fix one of the values to be unique.

Reply:
removed the condition, it's really not needed anymore